sdk/communication/azure-communication-phonenumbers/swagger/PHONE_NUMBER_SWAGGER.md

```diff
@@ -22,4 +22,18 @@ v3: true
 python: true
 title: Phone Numbers Client
 models-mode: msrest
+```
+
+### Removed Property error from PhoneNumberSearchResult
+``` yaml
+directive:
+  where-model: PhoneNumberSearchResult
+  remove-property: error
+```
+
+### Removed Property errorCode from PhoneNumberSearchResult
+``` yaml
+directive:
+  where-model: PhoneNumberSearchResult
+  remove-property: errorCode
 ```
```

Comment:
Could we add the same suppressions from the .NET SDK to prevent cherry picker generation? This is important because the generated client counts towards our coverage, but it is not used at all.

https://github.com/Azure/azure-sdk-for-net/blob/03d46a5e0a868fa42d4e25c2bd8065932a209e0e/sdk/communication/Azure.Communication.PhoneNumbers/src/autorest.md?plain=1#L44-L54